CMakeLists.txt

```diff
@@ -144,6 +148,10 @@ set(CM_PROJECTS_ALL)
 # Include all optional projects. These are the ones user can pick and choose.
 # Some of these allow using system libraries too.
 include(zlib)
+include(freetype)
+include(fontconfig)
```

Comment:
From what Cory says you probably want `elseif(UNIX AND NOT APPLE)` for fontconfig.